Enigmatry.Entry.Blueprint.Infrastructure.Tests/LoggerConfigurationExtensions.cs

```diff
@@ -0,0 +1,25 @@
+using System.Globalization;
+using Serilog;
+using Serilog.Events;
+
+namespace Enigmatry.Entry.Blueprint.Infrastructure.Tests;
+
+public static class LoggerConfigurationExtensions
+{
+    public static void ConfigureSerilogForIntegrationTests(this LoggerConfiguration loggerConfiguration) =>
+
+        // this allows serilog log statements to appear in the test console runner
+        // e.g. database sql queries (issued by DbContext) can be observed this way in the test
+        loggerConfiguration.WriteTo.Console(
+                LogEventLevel.Information,
```

Comment:
With Information log level we can see what is happening in our code. However some noise appears, which makes reading logs harder. The noise is supressed below.